packages/core-data/src/entity-types/term.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import type { Context, ContextualField, OmitNevers } from './helpers';
+
+import type { BaseEntityRecords as _BaseEntityRecords } from './base-entity-records';
+
+declare module './base-entity-records' {
+	export namespace BaseEntityRecords {
+		export interface Term< C extends Context > {
+			/**
+			 * A human-readable description of the term.
+			 */
+			description: ContextualField< string, 'view' | 'edit', C >;
+			/**
+			 * The title for the term.
+			 */
+			name: string;
+			/**
+			 * An alphanumeric identifier for the term.
+			 */
+			slug: string;
+			/**
+			 * The number of objects with the term.
+			 */
+			count: number;
```

Comment:
I'd suggest auditing all fields for context. `count` seems to be available for `view` and `edit` and not for `embed`.